src/content/app/genome-browser/components/drawer/drawer-views/variant-group-legend/VariantGroupLegend.tsx

```diff
@@ -0,0 +1,97 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+import variantGroups from './variantGroups';
+
+import { getBrowserActiveGenomeId } from 'src/content/app/genome-browser/state/browser-general/browserGeneralSelectors';
+
+import ExternalLink from 'src/shared/components/external-link/ExternalLink';
+
+import type { VariantLegendView } from 'src/content/app/genome-browser/state/drawer/types';
+import VariantGroupLegendColours from 'src/content/app/genome-browser/components/drawer/drawer-views/variant-group-legend/VariantGroupLegendColours';
+
+import styles from './VariantGroupLegend.scss';
+import { Table } from 'src/shared/components/table';
+
+type Props = {
+  drawerView: VariantLegendView;
+};
+
+const VariantGroupLegend = (props: Props) => {
+  const activeGenomeId = useSelector(getBrowserActiveGenomeId);
+
+  if (!activeGenomeId) {
+    return null;
+  }
+
+  const currentVariantGroup = variantGroups.find(
+    ({ label }) => label === props.drawerView.group
+  );
+  const legend = VariantGroupLegendColours.find(
+    (legend) => legend.id === currentVariantGroup?.id
+  );
+
+  if (!currentVariantGroup) {
+    return null;
+  }
+
+  return (
+    <div className={styles.container}>
+      <div className={styles.label}>
+        <span
+          className={styles.colorMarker}
```

Comment:
Inconsistent spelling here. Don't we usually use the British spelling for our own variables/names?